packages/outline-storybook/stories/components/outline-container.stories.ts

```diff
@@ -0,0 +1,272 @@
+import { html, TemplateResult } from 'lit';
+import { StoryFn } from '@storybook/web-components';
+
+import '@phase2/outline-alert';
+import '@phase2/outline-container';
+import '@phase2/outline-image';
+
+import code1 from '@phase2/outline-static-assets/media/tech/1440/code-1.jpg';
+import code2 from '@phase2/outline-static-assets/media/tech/1440/code-2.jpg';
+import code3 from '@phase2/outline-static-assets/media/tech/1440/code-3.jpg';
+import code4 from '@phase2/outline-static-assets/media/tech/1440/code-4.jpg';
+import code5 from '@phase2/outline-static-assets/media/tech/1440/code-5.jpg';
+import code6 from '@phase2/outline-static-assets/media/tech/1440/code-6.jpg';
+import code7 from '@phase2/outline-static-assets/media/tech/1440/code-7.jpg';
```

Comment:
suggestion: Remove this import, if the image is unused. This should fix the error below.